extension/extension/background.js

```diff
@@ -9,7 +9,7 @@ var RAW_DEBUG = false;
 /* This variable must be manually updated to specify the server that
  * the data will be sent to.  
 */
-var WEBSOCKET_SERVER_URL = "wss://learning-observer.org/wsapi/in/" 
```

Comment:
Why has this been removed?

Reply:
Do you mean the trailing whitespace? I often updated the server url and removed it the process when I noticed that there was a trailing whitespace. Was I wrong to have done that?